app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -21,6 +21,7 @@
     @BindView(R.id.contributionSequenceNumber) TextView seqNumView;
     @BindView(R.id.contributionProgress) ProgressBar progressView;
     @BindView(R.id.failed_image_options) LinearLayout failedImageOptions;
+    @BindView(R.id.failed_image_options_retry) LinearLayout failedImageOptionsretry;
 
```

Comment:
Ideally this PR should only edit .xml file. Since buttons are already defined and works fine. Please solve this issue by just editing .xml file, ie. adding a space between them